src/views/server_setup_views.py

```
@@ -0,0 +1,142 @@
+                label = "Medical Commands",
+                value = "medical_commands"
+            ),
+            discord.SelectOption(
+                label = "Mining Commands",
+                value = "mining_commands"
+            ),
+            discord.SelectOption(
+                label = "aUEC Tracking Commands",
+                value = "auec_commands"
+            ),
+            discord.SelectOption(
+                label = "Trading Commands",
+                value = "trading_commands"
+            )
+        ],
+        row = 0
+    )
+    async def callback(self, select, interaction: discord.Interaction):
+        """
+        The Select menu
+        """
+        if not self.selected_commands:
+            for value in select.values:
+                if value not in self.selected_commands:
+                    self.selected_commands.append(value)
+                else:
+                    self.selected_commands.remove(value)
+            interaction.response.send_message(f"Selected Commands: {self.selected_commands}")
+        else:
+            for value in select.values:
+                if value not in self.selected_commands:
+                    self.selected_commands.append(value)
+                else:
+                    self.selected_commands.remove(value)
+            interaction.response.edit_message(f"Selected Commands: {self.selected_commands}")
+            
+
+    @discord.ui.button(label = "Enable", style = discord.ButtonStyle.green, row = 1)
+    async def enable_button(self, button, interaction: discord.Interaction):
+        """
+        Confirm the selection of the above menu.
+        """
+        await interaction.response.edit_message(f"Confirmed Command Selection. You have selected: {self.selected_commands}")
+        try:
+            await slash_logic.prepare_commands_to_update(interaction.guild_id, self.selected_commands, True)
+            success = True
+        except Exception as e:
+            logger.error(e)
```

You should never just "catch all exceptions". Especially when you aren't actually handling anything, just printing the error to the log. Catch the exception that you excpect is possible to throw.